packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -58,33 +55,26 @@ export const Tabs: Primitive<TabsProps, typeof Flex> = ({
     value: currentIndex != null ? currentIndex.toString() : undefined,
     onValueChange: onChange,
   };
-
   return (
     <Root {...rootProps}>
       <List aria-label={ariaLabel}>
         <Flex
-          alignContent={alignContent}
-          alignItems={alignItems}
           className={classNames(ComponentClassNames.Tabs, className)}
-          direction={direction}
-          gap={gap}
-          justifyContent={justifyContent}
-          wrap={wrap}
           data-indicator-position={indicatorPosition}
+          ref={ref}
           {...rest}
         >
-          {tabs.map(({ className, isDisabled, title, ...rest }, index) => (
-            <RadixTab
-              className={classNames(ComponentClassNames.TabItems, className)}
-              data-spacing={spacing}
-              disabled={isDisabled}
-              key={index}
-              style={prefixer(convertStylePropsToStyleObj(rest))}
-              value={`${index}`}
-            >
-              {title}
-            </RadixTab>
-          ))}
+          {React.Children.map(children, (child, index) => {
+            if (!isTabsType(child)) {
+              return null;
+            }
+
+            return React.cloneElement(child, {
```

Comment:
By cloning, we are able to clone the `ref` passed along. I don't find a way to reference `ref` directly from `child`